scanner/mappers/mappers.go

```diff
@@ -242,7 +246,7 @@ func toProtoV4PackageVulnerabilitiesMap(ccPkgVulnerabilities map[string][]string
 	return pkgVulns
 }
 
-func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircore.Vulnerability) (map[string]*v4.VulnerabilityReport_Vulnerability, error) {
+func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircore.Vulnerability, _ []json.RawMessage) (map[string]*v4.VulnerabilityReport_Vulnerability, error) {
```

Comment:
nit:
```suggestion
func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircore.Vulnerability) (map[string]*v4.VulnerabilityReport_Vulnerability, error) {
```

Reply:
I foresee we will consume enrichments here, but we haven't done anything yet because no enrichments are being returned.